backend/src/relationships/blocks/dto/update-block.dto.ts

```diff
@@ -0,0 +1,7 @@
+import { OmitType } from '@nestjs/swagger';
+import { BlockDto } from './block.dto';
+
+export class UpdateBlockDto extends OmitType(BlockDto, [
+  'sourceId',
+  'targetId',
```

Comment:
```suggestion
export class UpdateBlockDto extends OmitType(BlockDto, [
  'id',
  'sourceId',
  'targetId',
```

I think the `id` should be omitted as well, to prevent the client from trying to change it